backend/src/utils/constants.ts

```diff
@@ -136,3 +136,6 @@ export const DEFAULT_NOTEBOOK_SIZES: NotebookSize[] = [
 
 export const imageUrlRegex =
   /^([\w.\-_]+((?::\d+|)(?=\/[a-z0-9._-]+\/[a-z0-9._-]+))|)(?:\/|)([a-z0-9.\-_]+(?:\/[a-z0-9.\-_]+|))(?::([\w.\-_]{1,127})|)/;
+
+export const THANOS_DEFAULT_RBAC_PORT = '9092';
+export const THANOS_DEFAULT_OAUTH_PORT = '9091';
```

Comment:
I know this is back and forth... but can we rename and add a comment?

```suggestion
// Won't work for basic users
export const THANOS_DEFAULT_ADMIN_PORT = '9091';
```